cj3.txt

```diff
@@ -61057,8 +61057,8 @@ nftwd  𩻧
 nftwv  𩼅
 nftyb  𩹩
 nftyj  𩺵
-nftyv  𩷶
 nftyu  𩺊
+nftyv  𩷶

```

Comment:
`sort-txt` does not account for character with various key mappings. The order of such characters is vague: only one of its key mappings (in this case it's `nftyv` for 𩺊) is used for sorting. Since the `sort-txt` does not reorder the URO and misc, I suggest we loose the policy for rare characters -- one can still find such mappings even if they are not grouped together in `cj3.txt`.